pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/GetCancerSuggestionsRouteStandaloneTest.java

```diff
@@ -184,7 +254,6 @@ public void givenLookupWithCyrillicCharacterThatDoesNotMatchAnything() {
     }
 
     private static class TestData {
-        public static final String STUDY_GUID = "test-study1";
```

Comment:
This test was using a phony study guid that did not exist in the database, which was causing me grief when trying to adjust the study's list of configured languages.  I've removed it and changed the test so that it uses a real study guid.

Reply:
`StudyLanguageResolutionFilter` doesn't throw errors if the study isn't real.  It was defaulting to English, and since `GetCancerSuggestionsRoute` doesn't do anything with the study guid, it worked.  Once I tried to update the study to support Spanish, things broke since the study didn't exist in the database.